### DISCUSSION TITLE
Feature: Display tracks for the last N days

### DISCUSSION THREAD

**Discussion by pricewatermouse** [osmandapp/OsmAnd#17369] [Ideas]
As a frequent traveler and user of OsmAnd, I want the application to automatically display tracks on the map for the last N days,
so that I can easily visualize my recent travel routes and avoid clutter on the map.

### Acceptance Criteria:

1. The user should have the option to specify the number of days ('N') for which they want to display tracks on the map.
2. By default, OsmAnd should display tracks for the last 7 days unless the user specifies a different value.
3. The user should have the ability to disable the automatic display of tracks if they prefer not to visualize their travel routes.
4. If the user wants to manually select other tracks to display on the map, he or she should have the ability to do so.
5. The style for the latest tracks should be visually distinct and easily distinguishable from current track and other tracks styles for clarity.
6. The application should provide a variety of style options for the latest tracks, allowing the user to customize their appearance.

**Comment by vshcherb** [comment]
We plan to have Filters for Tracks (similar to View in Sql databases) so probably after that we can make filters visible on the map
